lib/shopify_transporter/pipeline/magento/order/line_items.rb

```diff
@@ -26,13 +26,23 @@ def combine_associated_line_items(line_items)
             line_items.group_by { |line_item| line_item['sku'] }.map do |_, associated_items|
               case associated_items.size
               when 1
-                associated_items.first
+                associated_items.first.except('product_type')
               when 2
                 parent = associated_items.find { |x| x['product_type'] == 'configurable' }
                 child = associated_items.find { |x| x['product_type'] == 'simple' }
-                parent.merge(child.slice('name'))
-              end.except('product_type')
-            end
+                next line_items_without_type(associated_items) unless should_merge_child_into_parent?(parent, child)
+
+                next parent.merge(child.slice('name')).except('product_type')
```

Comment:
We should add en else case to handle potential cases where there somehow are 3 or more line items with the same sku. I think we can still combine them if the price is `0`, but we should at least return something.

Reply:
I added an `else` statement so that the code will return the line items without merging anything. I don't know if we should try to combine the items if any price is `0`. It's hard to figure out the merchant's original intention of more than 3 line item sharing the same SKU. Was it a mistake or was it intentional? Which line items should we merged into if there are multiple ones with different non-zero prices..? I feel like what I can do here is very limited  and cannot afford doing the wrong guess.

Reply:
Agreed. Definitely better to have more data than wrong data for these cases, until we know for sure.